src/components/organisms/connections/integrations/auth0/add.tsx

```diff
@@ -17,23 +19,74 @@ export const Auth0IntegrationAddForm = ({
 }) => {
 	const { t } = useTranslation("integrations");
 
-	const { handleOAuth } = useConnectionForm(oauthSchema, "create");
+	const { errors, handleCustomOauth, handleSubmit, isLoading, register } = useConnectionForm(
+		auth0IntegrationSchema,
+		"create"
+	);
 
 	useEffect(() => {
 		if (connectionId) {
-			handleOAuth(connectionId, Integrations.auth0);
+			handleCustomOauth(connectionId, "auth0");
```

Comment:
Why not use` Integrations.auth0` ?
```suggestion
			handleCustomOauth(connectionId, Integrations.auth0);
```